lib/chibi/math/prime.scm

```diff
@@ -75,20 +76,43 @@
 
 (define (modular-root-of-one? twos odd a n neg1)
   ;; Returns true iff any (modular-expt a odd*2^i n) for i=0..twos-1
-  ;; returns 1 modulo n.
+  ;; returns 1 or -1 modulo n.
```

Comment:
It's fine to delete this function if not used anymore.

Reply:
done